### ISSUE TITLE
plot_posterior() with point estimate = 'mode' fails

### ISSUE THREAD

**Issue by JWarmenhoven** [pymc-devs/pymc#1242]
Plotting posterior distributions for a trace using `plot_posterior()` fails with parameter `point_estimate='mode'`

``` python
pmc.plot_posterior(trace, varnames=['kappa', 'omega'], point_estimate='mode', figsize=(15,5));
```

``` python
NameError                                 Traceback (most recent call last)
<ipython-input-5-204472b2f7ba> in <module>()
----> 1 pmc.plot_posterior(trace, varnames=['kappa', 'omega'], point_estimate='mode', figsize=(15,5));

/Users/jordi/anaconda/envs/probalistic/lib/python3.5/site-packages/pymc3-3.0-py3.5.egg/pymc3/plots.py in plot_posterior(trace, varnames, transform, figsize, alpha_level, round_to, point_estimate, rope, ref_val, kde_plot, plot_transformed, ax, **kwargs)
    755         for a, v in zip(ax, varnames):
    756             tr_values = transform(trace.get_values(v, combine=True, squeeze=True))
--> 757             plot_posterior_op(tr_values, ax=a)
    758             a.set_title(v)
    759 

/Users/jordi/anaconda/envs/probalistic/lib/python3.5/site-packages/pymc3-3.0-py3.5.egg/pymc3/plots.py in plot_posterior_op(trace_values, ax)
    723         format_axes()
    724         display_hpd()
--> 725         display_point_estimate()
    726         if ref_val is not None:
    727             display_ref_val(ref_val)

/Users/jordi/anaconda/envs/probalistic/lib/python3.5/site-packages/pymc3-3.0-py3.5.egg/pymc3/plots.py in display_point_estimate()
    672                 point_text = '{}={}'.format(point_estimate, point_value.round(round_to))
    673             elif point_estimate == 'mode':
--> 674                 point_value = stats.mode(trace_values.round(round_to))[0][0]
    675                 point_text = '{}={}'.format(point_estimate, point_value.round(round_to))
    676             elif point_estimate == 'median':

NameError: name 'stats' is not defined 
```

It seems to work when I modify `plots.py` to use scipy.stats.mode instead.

**Comment by twiecki** [comment]
Thanks for reporting @JWarmenhoven, your fix looks correct, want to do a PR?

**Comment by JWarmenhoven** [comment]
ok. #1243

**Comment by twiecki** [comment]
Closed by #1243.
